src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { setAuthHeader } from './functions';
+export const get = async (url, params) => {
+    setAuthHeader();
+    const result = await axios.get(url, params);
+    return result.data;
+};
+export const post = async (url, params) => {
+    setAuthHeader();
+    const result = await axios.post(url, params);
+    return result.data;
+};
```

Comment:
can you explain this file when you get a chance? how / if it works with our backend -- just want to be aware

Reply:
Ah fantastic, i didn't actually end up using these functions. Essentially, every call to the api requires authorization tokens, so this ensure that they're in the request. I'm going to delete these because they're not used, but none of them should actually have anything to do with our backend, unless we need the backend to call to an api